File: ProjetoConsulta.Web/ClientApp/src/model/hospital.ts
import { Marcacao } from "./marcacao";

export class Hospital {

  id: number;
  nomeHospital: string;
  endereco: string;
  email: string;
  contato: string;
  marcacoes: Marcacao[];

  constructor(_id: number, _nomeHospital: string, _endereco: string, _email: string, _contato: string) {
    this.id = _id;
    this.nomeHospital = _nomeHospital;
    this.endereco = _endereco;
    this.email = _email;
    this.contato = _contato;


  }
}

File: ProjetoConsultaPaciente.Dominio/Entidade/Hospital.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ProjetoConsultaPaciente.Dominio.Entidade
{
    public class Hospital
    {
        public System.Guid Id { get; set; }
        public string Nome { get; set; }
        public string Endereco { get; set; }
        public string Email { get; set; }
        public string Contato { get; set; }

        public virtual ICollection<Marcacao> Marcacoes { get; set; }
    }
}
